xla/service/cpu/onednn_util.h

```diff
@@ -23,14 +23,40 @@ limitations under the License.
 #include "unsupported/Eigen/CXX11/Tensor"
 #include "dnnl.hpp"
 #include "xla/hlo/ir/hlo_instruction.h"
+#include "xla/service/cpu/runtime_lightweight_check.h"
 #include "xla/service/cpu/backend_config.pb.h"
 #include "xla/tsl/util/onednn_threadpool.h"
 #include "xla/xla_data.pb.h"
 #include "tsl/platform/cpu_info.h"
+#include "xla/service/cpu/backend_config.pb.h"
 
 namespace xla {
 namespace cpu {
 
+using dnnl::engine;
+using dnnl::memory;
```

Comment:
Per [Google C++ Style Guide](https://google.github.io/styleguide/cppguide.html#Aliases), we don't use using aliases or typedefs in header files just for convenience. (Only when they are intended to be part of the API.) 

Please use the full type instead. (Can still use using aliases in .cc file or private scopes.)

Reply:
Fixed.